pkg/authorization/api/validation/validation.go

```diff
@@ -25,10 +25,36 @@ func ValidateSubjectAccessReview(review *authorizationapi.SubjectAccessReview) f
 func ValidateResourceAccessReview(review *authorizationapi.ResourceAccessReview) fielderrors.ValidationErrorList {
 	allErrs := fielderrors.ValidationErrorList{}
 
-	if len(review.Verb) == 0 {
+	if len(review.Action.Verb) == 0 {
 		allErrs = append(allErrs, fielderrors.NewFieldRequired("verb"))
 	}
-	if len(review.Resource) == 0 {
+	if len(review.Action.Resource) == 0 {
+		allErrs = append(allErrs, fielderrors.NewFieldRequired("resource"))
+	}
+
+	return allErrs
+}
+
+func ValidateLocalSubjectAccessReview(review *authorizationapi.LocalSubjectAccessReview) fielderrors.ValidationErrorList {
+	allErrs := fielderrors.ValidationErrorList{}
+
+	if len(review.Action.Verb) == 0 {
+		allErrs = append(allErrs, fielderrors.NewFieldRequired("verb"))
+	}
+	if len(review.Action.Resource) == 0 {
+		allErrs = append(allErrs, fielderrors.NewFieldRequired("resource"))
+	}
+
```

Comment:
seems like authorizationattributes.namespace should exist and match objectmeta.namespace, and possibly default to the objectmeta.namespace. I want to make sure someone can't submit a localsubjectaccessreview to "foo" with a namespace field set to "bar"